README.md

```diff
@@ -48,9 +48,10 @@ OLLAMA_HOST=0.0.0.0 OLLAMA_ORIGINS=* ollama serve
 
 ### Using Docker 🐳
 
-```bash
-docker build --build-arg OLLAMA_API_BASE_URL='http://localhost:11434/api' -t ollama-webui .
```

Comment:
Better to be explicit rather than implicit, so that users have to reference less documentation.

It _looks_ "cool" to copy and paste fewer characters, but it's much more frustrating to have to reference documentation for things that aren't intuitive.

That said, now that we know that `''` will work correctly, showing `OLLAMA_API_BASE_URL=''` would both be fewer characters and still be mostly self-documenting.

Reply:
Agreed!